service/templates/admin.html

```diff
@@ -449,14 +449,23 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 												</td>
 											</tr>
 											<tr>
-												<td>Use Static API credentials for users </td>
+												<td>Allow Static API credentials for users </td>
 												<td colspan="2">
 													<md-radio-group data-ng-model="Settings.service_flags.static_api_keys_disabled" layout="row">
 														<md-radio-button data-ng-value="false"></md-radio-button>
 														<md-radio-button data-ng-value="true"></md-radio-button>
 													</md-radio-group>
 												</td>
 											</tr>
+											<tr>
+												<td>Use JWT token for CLI </td>
+												<td colspan="2">
+													<md-radio-group data-ng-model="Settings.service_flags.use_jwt_token_for_cli" layout="row">
+														<md-radio-button data-ng-value="false"></md-radio-button>
```

Comment:
All the feature flags are default "enabled" but this flag is a default "disabled" flag - you would have to flip the two radio buttons true/false to account for  this.

Reply:
It's not being used for anything yet so I don't think it should be a problem.

Reply:
It's similar to "Use Evergreen for Test Results Service" and "Use elastic IP features for task hosts"

Reply:
I see what you mean. I updated it to follow that pattern